_typos.toml

```diff
@@ -0,0 +1,2 @@
+[default]
+extend-ignore-re = ["\\(\\[([a-z0-9]{7})\\]\\("]
```

Comment:
The regex `"\(\\[([a-z0-9]{7})\\]\\("` matches beyond hexadecimal ranges. For accuracy, use `"\(\\[([a-fA-F0-9]{7})\\]\\("` to strictly match 7-character hex hashes in markdown links, enhancing specificity and reducing mismatches.